main.py

```diff
@@ -1,5 +1,59 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Genre, Actor
+
 
 def main():
-    pass
+    Genre.objects.create(
+        name="Western"
+    )
+    Genre.objects.create(
+        name="Action"
+    )
+    Genre.objects.create(
+        name="Dramma"
+    )
+    Actor.objects.create(
+        first_name="George",
+        last_name="Klooney"
+    )
+    Actor.objects.create(
+        first_name="Kianu",
+        last_name="Reaves"
+    )
+    Actor.objects.create(
+        first_name="Scarlett",
+        last_name="Keegan"
+    )
+    Actor.objects.create(
+        first_name="Will",
+        last_name="Smith"
+    )
+    Actor.objects.create(
+        first_name="Jaden",
+        last_name="Smith"
+    )
+    Actor.objects.create(
+        first_name="Scarlett",
+        last_name="Johansson"
+    )
+    Genre.objects.filter(
+        name="Dramma"
+    ).update(name="Drama")
+    Actor.objects.filter(
+        last_name="Klooney"
```

Comment:
You need filtered by first_name too

Reply:
Yes, indeed. My code works for this example, but in case of Clooney family, I would've gotten a mistake. Thanks.